cmd/faInfo/faInfo.go

```diff
@@ -0,0 +1,82 @@
+// Command Group: "FASTA and Multi-FASTA Tools"
```

Comment:
It is hard for me to see the code changes.  Is this a re-write for a renaming of the file?  In this case the logic is simple enough that people can review the whole program as new, but in the future it may be quicker for reviewers if we use "git mv" to rename the file and then make the desired edits.

Reply:
I didnt realize it was going to do this, and totally agree that it is hard to read. I will be sure to use the git mv functionality in the future.